spec/orbital/request_spec/cvv_indicator_spec.rb

```diff
@@ -0,0 +1,164 @@
+require 'spec_helper'
```

Comment:
If you put them in a new subdirectory, the `Rakefile` needs to be updated or tests won't be run. Also, the expectation (e.g. for Travis) is that `rake` runs the fast tests.